collection-scripts/gather_crs

```diff
@@ -1,59 +1,41 @@
 #!/bin/bash
 
-unset KUBECONFIG
-for localns in $(/usr/bin/oc get migrationcontrollers.migration.openshift.io --all-namespaces --no-headers | awk '{print $1}'); do
```

Comment:
I deduped all code in sub-gather files that collects migrationcontrollers. We only need to do this once from the main `gather`.